jstp/src/main/java/com/metarhia/jstp/connection/AbstractSocket.java

```diff
@@ -10,34 +13,97 @@
    */
   boolean connect();
 
+  /**
+   * Sends the message
+   *
+   * @param message message
```

Comment:
We should avoid such comments, at least let it be 'message the string to be sent'